docs/user-guide/custom-themes.md

```diff
@@ -6,12 +6,15 @@ A guide to creating and distributing custom themes.
 
 !!! Note
 
-    If you are looking for third party themes, they are listed in the MkDocs
-    [community wiki](https://github.com/mkdocs/mkdocs/wiki/MkDocs-Themes). If
-    you want to share a theme you create, you should list it on the Wiki.
+If you are looking for third party themes, they are listed in the
+[MkDocs community wiki](https://github.com/mkdocs/mkdocs/wiki/MkDocs-Themes). If 
```

Comment:
The trailing white-space here is causing the [MarkdownLint](https://travis-ci.org/mkdocs/mkdocs/jobs/163943352#L205) test to fail, as does the [long line](https://travis-ci.org/mkdocs/mkdocs/jobs/163943352#L206).